awkward-cpp/CMakeLists.txt

```
@@ -11,6 +11,17 @@ project(
 message(STATUS "CMake version ${CMAKE_VERSION}")
 message(STATUS "CMAKE_BUILD_TYPE = ${CMAKE_BUILD_TYPE}")
 
+# Check for header-only libraries
+if(NOT (EXISTS "header-only" AND EXISTS "include/awkward/kernels.h"))
+  message(
+    FATAL_ERROR
+      "\
+awkward-cpp relies upon generated and copied artefacts such as the header-only libraries and generated kernel headers.\
+These could not be found, which indicates that the preparation step for building this package was skipped.\
+Please check CONTRIBUTING.md to learn more about this process.\
```

This clickable URL (some terminals make URLs clickable) takes a user to the installation instructions, which in turn links to CONTRIBUTING.md for "more details."

But also, it includes the missing line to triple-click and middle-click to copy and paste on the spot. Sometimes I run commands that I know will fail because they'll give me the line I was supposed to type first, and and if they're easy enough to select and copy-paste, then I don't even need to type them.

```suggestion
awkward-cpp relies upon generated and copied artefacts such as the header-only libraries and generated kernel headers. \
These could not be found, which indicates that\n\n\
    nox -s prepare\
\n\nwas skipped or failed. \
Please check https://github.com/scikit-hep/awkward#installation-for-developers to learn more about this process.\
```

---

I'm hoping that it gets rendered as

```
awkward-cpp relies upon generated and copied artefacts such as the header-only libraries and generated kernel headers. These could not be found, which indicates that

    nox -s prepare

was skipped or failed. Please check https://github.com/scikit-hep/awkward#installation-for-developers to learn more about this process.
```

to make the copy-pasteable line selectable with a triple-click.